xrpl_api/src/api/account_info.rs

```diff
@@ -10,8 +10,6 @@ pub struct AccountInfoRequest {
     pub queue: Option<bool>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub signer_lists: Option<bool>,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub strict: Option<bool>,
```

Comment:
Removed this parameter since I cannot find it in the documentation, I don't think it exists?